src/models/token.js

```diff
@@ -0,0 +1,31 @@
+const { DataTypes } = require('sequelize');
+
+const { User } = require('./user');
+const { sequelize } = require('../utils/db.js');
+const { getExpireTime } = require('../utils/getExpireTime.js');
+
+const MONTH_IN_MINUTES = 30 * 24 * 60;
+
+const Token = sequelize.define(
+  'Token',
+  {
+    refreshToken: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    expireAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: getExpireTime(MONTH_IN_MINUTES),
```

Comment:
Consider using Sequelize's built-in date manipulation methods for `expireAt` default value to ensure compatibility across different database systems.